.circleci/config.yml

```diff
@@ -174,6 +174,8 @@ commands:
           command: |
             if [[ $CIRCLE_TAG == android-v* ]]; then
               sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${CIRCLE_TAG:9}/" gradle.properties
+            elif [[ $CIRCLE_TAG == android-auto-extension-v* ]]
```

Comment:
Would like thoughts/feedback for the name of the Android Auto extension release tag cc @mapbox/maps-android

Reply:
another idea would be use `extension-androidauto-v*`, so that it matches the module name, and in the future if we publish separate version for other extensions, these extensions can be grouped together, wdyt?

Reply:
Aligning tag prefix and the module name seems even better indeed. Not sure I follow the grouping though, do you mean multiple extensions could be published separately with the same tag?

Reply:
I meant that users could group the extensions/plugins and sdk releases more easily, as they have different prefixes.

Reply:
Changed to `extension-androidauto-v*`